clicache/src/CacheFactory.cpp

```diff
@@ -70,14 +71,12 @@ namespace Apache
           pdxReadSerialized = nativeCache->getPdxReadSerialized();
 
           Log::SetLogLevel(static_cast<LogLevel>(native::Log::logLevel( )));
-          native::createAppDomainContext = &Apache::Geode::Client::createAppDomainContext;
-
 
           DistributedSystem::ManagedPostConnect(cache);
           DistributedSystem::registerCliCallback();
           auto&& cacheImpl = CacheRegionHelper::getCacheImpl(nativeCache.get());
           cacheImpl->getSerializationRegistry()->setPdxTypeHandler(new ManagedPdxTypeHandler());
-		  cacheImpl->getSerializationRegistry()->setDataSerializableHandler(new ManagedDataSerializableHandler());
+		      cacheImpl->getSerializationRegistry()->setDataSerializableHandler(new ManagedDataSerializableHandler());
```

Comment:
Looks like formatting got out of wack with your Visual Studio. Can you back out the change on this lineZ or better yet make the indent match the rest of the file:

Reply:
Fixed to match rest of file.